test/e2e-1of2/mcd_test.go

```
@@ -1305,3 +1306,170 @@ func parseMetricValue(metricLine string) (int, error) {
+	msName := fmt.Sprintf("%s-e2e-fbfail", poolName)
+
+	clonedMS := originalMS.DeepCopy()
+	clonedMS.Name = msName
+	clonedMS.ResourceVersion = ""
+	clonedMS.UID = ""
+	var zero int32
+	clonedMS.Spec.Replicas = &zero
+	clonedMS.Spec.Selector.MatchLabels["machine.openshift.io/cluster-api-machineset"] = msName
+	clonedMS.Spec.Template.Labels["machine.openshift.io/cluster-api-machineset"] = msName
+
+	clonedMS, err = machineclient.MachineV1beta1().MachineSets(ctrlcommon.MachineAPINamespace).Create(ctx, clonedMS, metav1.CreateOptions{})
```

golangci-lint (2.12.2)

[error] SA4006: this value of clonedMS is never used

(staticcheck)